### ISSUE TITLE
Have some bugs with date displaying

### ISSUE THREAD

**Issue by deni1s** [andrewjapar/rangedatepicker#3]
I init calendar like this:   
```
val startDate = Calendar.getInstance(TimeZone.getDefault(), Locale.getDefault())
        startDate.set(2019, 9, 1)
        val endDate = Calendar.getInstance(TimeZone.getDefault(), Locale.getDefault()) // Today is 24.01.2020. Truly, does not matter which date I set

        calendar_view.apply {
            setMode(CalendarPicker.SelectionMode.RANGE) // You can set it via XML
            setRangeDate(startDate.time, endDate.time)
        }
```
And my phone display calendar like this:
![photo_2020-01-24_10-34-23](https://user-images.githubusercontent.com/10938186/73055280-1fa24980-3e95-11ea-9cf6-0c40e1ff0e11.jpg)
Xiaomi mi6, Android 9

**Comment by andrewjapar** [comment]
Hi @deni1s, currently this issue is on progress then i will notify you once it's resolved.

Thanks for reporting 👯‍♂

**Comment by andrewjapar** [comment]
This bug has been resolved on PR #5 

Thanks @deni1s
